bootstrap.sh

```diff
@@ -73,6 +73,9 @@ install_app recommendations
 install_app files_pdfviewer
 install_app profiler
 install_app hmr_enabler
+for app in $@; do
```

Comment:
Could you maybe extract that into a variable so we can adapt only the extraction bit in case of adding more cli flags?